### ISSUE TITLE
[NewTemplateRequest]

### ISSUE THREAD

**Issue by ankit-rathi** [microsoft/botframework-components#1241]
Need template to create Empty Bot

**Comment by benbrown** [comment]
Are you not able to see the empty bot template in Composer? There should be one...

Do you have access to NPM?

**Comment by ankit-rathi** [comment]
> Are you not able to see the empty bot template in Composer? There should be one...
> 
> Do you have access to NPM?

Yes I have access to NPM, still I son't see empty bot template in Composer...

**Comment by benbrown** [comment]
The list of templates are loaded when you first launch Composer. I know it has been multiple days, but have you restarted the application yet?

Also, can you tell me what operating system you are using, and what version of Composer you have installed?

**Comment by pavolum** [comment]
The bug fix for this is being tracked here with a potential current work around
https://github.com/microsoft/BotFramework-Composer/issues/8525
